ppl/zqd/handlers.go

```diff
@@ -606,3 +608,12 @@ func extractSpaceID(c *Core, w http.ResponseWriter, r *http.Request) (api.SpaceI
 	}
 	return api.SpaceID(id), true
 }
+
+func handleIdentityGet(c *Core, w http.ResponseWriter, r *http.Request) {
+	ident, ok := IdentifyFromContext(r.Context())
+	if !ok {
+		respondError(c, w, r, zqe.ErrNoCredentials())
```

Comment:
Wouldn't it be impossible to get this error given that `handleIdentityGet` is wrapped in core.authhandle? Or perhaps I am missing something.

Also wouldn't `zqe.InValid("no credentials")` be a better response code? If they got here it means they are authorized, its just an invalid request.